sdk/src/main/java/io/dapr/actors/runtime/ActorRuntime.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import java.util.HashMap;
+import java.util.Set;
+import java.util.function.Function;
+
+import io.dapr.actors.*;
+
+/**
+ * Contains methods to register actor types. Registering the types allows the runtime to create instances of the actor.
+ */
+public class ActorRuntime {
+    private static volatile ActorRuntime instance;
+    private static AppToDaprAsyncClient appToDaprHttpAsyncClient;
+    private final String TraceType = "ActorRuntime";
+    private final HashMap<String, ActorManager> actorManagers;
+
+    private ActorRuntime() throws IllegalStateException{
+        if (instance != null) {
+            throw new IllegalStateException("ActorRuntime should only be constructed once");
+        }
+
+        this.actorManagers = new HashMap<String, ActorManager>();
+        appToDaprHttpAsyncClient = new DaprClientBuilder().buildAsyncClient();
+    }
+
+    /**
+     * Returns an ActorRuntime object.
+     * @return An ActorRuntime object.
+     */
+    public static ActorRuntime getInstance() {
```

Comment:
Good standard way to do singleton. Since this is public, it makes sense. Let's try to minimize calls to this method in our code and get ActorRuntime via dependency injection as much as possible. LGTM here.